### PR TITLE
Fix: Remove trailing whitespace if value is empty

### PR DESCRIPTION
**Pull request by OskarStark**
This is the case if someone wants to add a blank line.

```diff
# IMPORTANT: Comment
-# 
+#
# FOO_BAR=baz
```

### CODE REVIEW

File: src/Configurator/EnvConfigurator.php [hunk 1]
```diff
@@ -51,6 +51,12 @@ private function configureEnvDist(Recipe $recipe, $vars, bool $update)
             foreach ($vars as $key => $value) {
                 $value = $this->evaluateValue($value);
                 if ('#' === $key[0] && is_numeric(substr($key, 1))) {
+                    if (empty($value)) {
```
Comment (fabpot) [inline src/Configurator/EnvConfigurator.php:54]: `if ('' === $value) {` then `$data .= "#\n";`?

**Review by fabpot — CHANGES_REQUESTED** [review]

File: src/Configurator/EnvConfigurator.php [hunk 2]
```diff
@@ -51,6 +51,12 @@ private function configureEnvDist(Recipe $recipe, $vars, bool $update)
             foreach ($vars as $key => $value) {
                 $value = $this->evaluateValue($value);
                 if ('#' === $key[0] && is_numeric(substr($key, 1))) {
+                    if ('' === $value) {
+                        $data .= "#\n";
+
+                        continue;
```
Comment (fabpot) [inline src/Configurator/EnvConfigurator.php:57]: Can we remove that "continue" and do an else to avoid the double-continue? Sorry for the nitpick

Comment (OskarStark) [reply src/Configurator/EnvConfigurator.php:57]: sure, no problem

File: src/Configurator/EnvConfigurator.php [hunk 3]
```diff
@@ -51,7 +51,11 @@ private function configureEnvDist(Recipe $recipe, $vars, bool $update)
             foreach ($vars as $key => $value) {
                 $value = $this->evaluateValue($value);
                 if ('#' === $key[0] && is_numeric(substr($key, 1))) {
-                    $data .= '# '.$value."\n";
+                    if ('' === $value) {
```
Comment (OskarStark) [inline src/Configurator/EnvConfigurator.php:54]:
```suggestion
                    if ('' === trim($value)) {
```

WDYT @fabpot ?

Comment (nicolas-grekas) [reply src/Configurator/EnvConfigurator.php:54]: this won't happend IRL, and if it does, I would trust the manifest did it on purpose

Comment (OskarStark) [reply src/Configurator/EnvConfigurator.php:54]: OK, ready to merge then

**Review by chalasr — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @OskarStark.

**Pull request merged**